routers/web/admin/hooks.go

```diff
@@ -59,9 +59,9 @@ func DefaultOrSystemWebhooks(ctx *context.Context) {
 	ctx.HTML(http.StatusOK, tplAdminHooks)
 }
 
-// DeleteDefaultOrSystemWebhook handler to delete an admin-defined system or default webhook
-func DeleteDefaultOrSystemWebhook(ctx *context.Context) {
-	if err := webhook.DeleteDefaultSystemWebhook(ctx, ctx.FormInt64("id")); err != nil {
+// handler to delete an admin-defined system or default webhook
+func DeleteWebhook(ctx *context.Context) {
+	if err := webhook.DeleteAdminWebhook(ctx, ctx.FormInt64("id")); err != nil {
```

Comment:
This is ignoring `:configType` at the moment: it's possible to `DELETE /api/v1/admin/hooks/system/9` when you should use `DELETE /api/v1/admin/hooks/default/9`.